code/ui-utils/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+}
+
+val mavenCoreVersion: String by project
+
+
+android {
+    namespace = "com.adobe.marketing.mobile.ui_utils"
```

Comment:
Can we keep the module name and namespace consistent? Also can we call this `aep-ui-utils` to avoid collisions with other ui libraries?